### ISSUE TITLE
Insufficient feedback during installation

### ISSUE THREAD

**Issue by jeremypw** [elementary/appcenter#1787]
### What Happened?

I inadvertantly started an "Update All" process while having Synaptic open.  The process proceeded to certain point but then stalled with the progressbar at 100%, the spinner working, and the cancel button tooltip reading "Waiting".  It was not immediate obvious what was causing the wait.  Once Synaptic was closed the process completed quickly.

### Steps to Reproduce

1. Start Synaptic or similar process
2. Launch AppCenter and start an "Update All" process
3. Observe that the process proceeds for a while then stalls with no obvious feedback as to reason.

### Expected Behavior

Clear user feedback should be given when an update process cannot complete until a user action is taken

### OS Version

6.x (Odin)

### Software Version

Compiled from git

**Comment by danrabbit** [comment]
Yeah if we can't explicitly get into about a package lock, we could use a timeout like mutter does for windows that aren't responsive, but either way we should probably throw a dialog with like "cancel" and "keep waiting" and some kind of copy about making sure other package managers are closed
